### DISCUSSION TITLE
Argo Workflow can only create 10 running tasks.

### DISCUSSION THREAD

**Discussion by LingClassmate** [argoproj/argo-workflows#9110] [Q&A] [answered]
What happened/what you expected to happen?

Argo Workflow can only create 10 running tasks. If you submit them again, you will be prompted `Pending: Workflow processing has been postponed because too many workflows are already running`

What version are you running?
`v3.3.5`

**Comment by terrytangyuan** [comment, marked as answer]
You can change parallelism in controller’s configmap.

**Reply by ni-todo-spot** [reply]
Can u please refer to an example / documentation so I can make sure what setting you are referring to 😃 ?

**Reply by terrytangyuan** [reply]
https://github.com/argoproj/argo-workflows/blob/master/docs/workflow-controller-configmap.yaml
```
  # Parallelism limits the max total parallel workflows that can execute at the same time
  # (available since Argo v2.3). Controller must be restarted to take effect.
  parallelism: 10
```

**Reply by ni-todo-spot** [reply]
perfect, thank you :)

**Reply by Guillermogsjc** [reply]
hi @terrytangyuan , just a question, which is  the default value for parallelism that workflow-controller assumes if not patched the `workflow-controller-configmap` with `parallelism` ? 

does this default limit value experimented a change in `v3.4` ?

Also, realised in another thread that data.namespaceParallelism can conflict with data.parallelism which one takes priority? Looking at manifests in quick manifests you are mainly using namespaceParallelism only, but would want to confirm this behaviour. Checked throttler.go but could not infer nothing :(

Currently having a workflow controller with only 10 tasks at a time (trying to scape from this limit), but there is no combination of those two parameters (put or not then in configmap), that scapes from this bound.
